content/contributing/opencue/Getting started/getting-started-linux.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Setting up a development environment on Linux"
+linkTitle: "Setting up a development environment on Linux"
+date: 2020-03-24
+weight: 1
+description: >
+  Set up your development environment on Linux
+---
+
+This is a guide to setting up a development environment on Linux that lets
+you run the entire OpenCue system, including Cuebot, CueGUI, and RQD. After
+you complete the setup, you can make changes to any part of the development
+environment.
+
+## Before you begin
+
+First, clone the
+[OpenCue git repository](https://github.com/AcademySoftwareFoundation/OpenCue)
+to your machine. You also need to download and install OpenCue's core
+dependencies:
+
+- [PostgreSQL](https://www.postgresql.org/download/linux/ubuntu/) version 9 or
+  greater.
+  - We recommend installing Postgres using a `yum`-based Linux distribution, such as
+  [CentOS](https://www.centos.org/):
+
+   1. First, run `yum` to install the required Postgres packages:
+
+       ```shell
+       yum install postgresql-server postgresql-contrib
+       ```
+
+   1. Next, initialize your Postgres installation and configure it to run as a
+    service:
+
+       ```shell
+       postgresql-setup initdb
+       systemctl enable postgresql.service
+       systemctl start postgresql.service
+       ```
+
+   1. Create a superuser named after your current OS user, which is used for the
+    rest of the admin commands in this guide.
+
+       ```shell
+       su -c "createuser -s $USER" postgres
+       ```
+- [Python 3.x](https://www.python.org/downloads/) 
```

Comment:
Are Python 3 and Java 11+ available by default on CentOS now? If they aren't we should add `yum` commands for them as well.

Reply:
Yes Python 2 is present by default in CentOS but it is EOL on January 1, 2020. So, we have to install Python 3 manually.
so, I will add both installation on CentOS manually through yum commands.